code/modules/mob/living/brain/brain_item.dm

```diff
@@ -146,3 +146,15 @@
 	desc = "We barely understand the brains of terrestial animals. Who knows what we may find in the brain of such an advanced species?"
 	icon_state = "brain-x"
 	origin_tech = "biotech=6"
+
+// IPC brain fuckery.
+
+/obj/item/organ/brain/ipc
+	name = "IPC posibrain"
+	slot = "brain"
+	desc = "A cube of shining metal, three inches to a side and covered in shallow grooves. This particular model seems specific to IPCs, due to lack of foresight. It will need an MMI adapter to be slotted into cyborgs or AI cores."
```

Comment:
If its a 3inch cube the size should be reduced.

It should probably be made to be of comparable size to regular brains, updating size & description accordingly.